project_national/sdr_upgrades_tmy3.yml

```diff
@@ -1149,4 +1149,24 @@ upgrades:
       - *attic_insulation_IECC_CZ1A
       - *attic_insulation_IECC_CZ2A-2B-3A-3B-3C
       - *attic_insulation_IECC_CZ4A-4B-4C-5A-5B-6A-6B-7A-7B
-      - *infiltration_30pct_reduction
\ No newline at end of file
+      - *infiltration_30pct_reduction
+
+  - upgrade_name: EV Ownership + L1 Charger
+    package_apply_logic:
+      - Electric Vehicle Ownership|No  
```

Comment:
Do we want to exclude vacant units?

Reply:
Addressed: [7647e49](https://github.com/NREL/resstock/pull/1362/commits/7647e4952709a4f8aa37ab515ba61cc9162da7a7)